transports/README.md

```
@@ -80,33 +80,43 @@ curl -L -o Dockerfile https://raw.githubusercontent.com/maximhq/bifrost/main/tra
 2. Build the Docker image:
 
 ```bash
-docker build \
-  --build-arg CONFIG_PATH=./config.json \
-  --build-arg PORT=8080 \
-  --build-arg POOL_SIZE=300 \
-  -t bifrost-transports .
+docker build -t bifrost-transports .
 ```
 
 3. Run the Docker container:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)